test/program-es6-imports.test.ts

```diff
@@ -8,7 +8,7 @@ import semver from 'semver';
 const entryFile = `${__dirname}/programs/es6-imports/cli/entry.js`;
 
 // Don't run the normal tests if older than Node.js version 12
-if (semver.lt(semver.clean(process.version), '13.2.0')) {
+if (semver.lt(semver.clean(process.version) ?? '', '13.2.0')) {
```

Comment:
I'm not sure what `semver.lt()` will make of an empty string.  It's probably going to be equivalent of 0, but it might be more directly comparable if the fallback value is set to '0.0.0' instead of ''.

Reply:
That seems reasonable.

```suggestion
if (semver.lt(semver.clean(process.version) ?? '0.0.0', '13.2.0')) {
```